src/components/confirmationPopup.js

```diff
@@ -0,0 +1,28 @@
+import Popup from "./Popup";
+
+class confirmationPopup extends Popup {
+  constructor(popupSelector,  ) {
+    super({ popupSelector });
+  }
+  close(){
+
+    super.close();
+
+  }
+  open() {
+    super.open();
+  }
+  setSubmitAction(submitHandler){
+    this.submitHandler = submitHandler;
+
+  }
+  setEventListeners() {
+    super.setEventListeners()
+    const saveButton = this._popupElement.querySelector(".modal__save-button");
+    saveButton.classList.add("transparent");
```

Comment:
* The confirmation button looks disabled https://disk.yandex.ru/i/nZvHFpYLnGoFHg but it should be **black** according to the design